che-eclipse-license-resource-bundle/src/main/resources/license-header.txt

```diff
@@ -1,8 +1,6 @@
 Copyright (c) ${h_license_years} ${h_copyrightOwner}
-All rights reserved. This program and the accompanying materials
-are made available under the terms of the Eclipse Public License v1.0
-which accompanies this distribution, and is available at
-http://www.eclipse.org/legal/epl-v10.html
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v2.0
+which is available at http://www.eclipse.org/legal/epl-2.0.html
 
-Contributors:
```

Comment:
Are you sure that we need to remove `Contributors` section?

Reply:
So, looks like `Contributors` doesn't suite for epl v2.0, so since the difference between `license-header.txt` `license-header2.txt` is a different `Contributors` section. `license-header2.txt` may be removed at all. WDYT?